<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_page</class>
 <widget class="QMainWindow" name="main_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,6,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="all_button">
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>all</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="group_button">
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>groups</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="channel_button">
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>channels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="users_button">
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>users</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="network_status"/>
        </item>
        <item>
         <widget class="QListWidget" name="member_list">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="lineWidth">
           <number>10</number>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" rowminimumheight="1,1">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="create_channel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>create channel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="join_channel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>join channel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="new_user">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>new user</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="create_group">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>create group</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="join_group">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>join group</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="12,1">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>817</width>
             <height>616</height>
            </rect>
           </property>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>290</y>
              <width>278</width>
              <height>214</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="new_message_text_edit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Berlin Sans FB Demi'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Berlin SansFB Demi'; color:#3ddbff;&quot;&gt;message&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="send_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Berlin Sans FB Demi</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
